views/index.html

```diff
@@ -20,6 +20,8 @@
 
     <!-- APPLICATION SCRIPTS -->
     <script src="scripts/app.js"></script>
+    <script src="scripts/controllers/AlbumsIndexController.js"></script>
```

Comment:
Let's talk about when to implement this pattern.  I think it's good but let's talk sequence.